ci/patch_src_in_place

```diff
@@ -3,6 +3,12 @@
 # shellcheck disable=SC2046,SC2035
 codespell -w --ignore-words-list nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden $(git ls-tree --full-tree --name-only HEAD)
 
+# The return code of codespell is the number of works it could not correct
+# because of multiple options.  We could report on these but they're rare
+# so just save the exit code here and use it later on.  They are reported
+# in the output of the CI job, just will not be commented on in github.
+RC=$?
+
```

Comment:
We really should be running this script with `set -e` enabled in which case this could/should be (defensively) re-written as:

```suggestion
RC=0
if ! codespell -w --ignore-words-list nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden $(git ls-tree --full-tree --name-only HEAD); then
    # The return code of codespell is the number of works it could not correct
    # because of multiple options.  We could report on these but they're rare
    # so just save the exit code here and use it later on.  They are reported
    # in the output of the CI job, just will not be commented on in github.
    RC=$?
```

Reply:
I left this because I'm still working on the script in other PRs and I don't want to change it to much to get the fix in for @frostedcmos

Reply:
So you will fix this in one of the other PRs?

Reply:
I'll add you and we can work out what's best then.